compiler/native/parse.go

```diff
@@ -35,34 +35,35 @@ func (c *client) Parse(v interface{}) (*types.Build, error) {
 		// parse string as yaml configuration
 		return ParseString(v)
 	default:
-		return nil, fmt.Errorf("unable to parse yaml: unrecognized type %T", v)
+		return nil, nil, fmt.Errorf("unable to parse yaml: unrecognized type %T", v)
 	}
 }
 
 // ParseBytes converts a byte slice to a yaml configuration.
-func ParseBytes(b []byte) (*types.Build, error) {
+func ParseBytes(b []byte) (*types.Build, []byte, error) {
 	config := new(types.Build)
 
 	// unmarshal the bytes into the yaml configuration
 	err := yaml.Unmarshal(b, config)
 	if err != nil {
-		return nil, fmt.Errorf("unable to unmarshal yaml: %v", err)
+		fmt.Fprintln(os.Stderr, yaml.FormatError(err, true, true))
```

Comment:
Is this supposed to be here?

Looks like only `ParseBytes()` will output an error to `os.Stderr` based off what I'm reading.